pkg/cmd/util/resource/visitor.go

```diff
@@ -0,0 +1,64 @@
+package resource
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
```

Comment:
There are multiple yaml packages in kusion, using the same is better.
And why you use the yaml official package "gopkg.in/yaml.v3"

Reply:
crd resources might be in YAML or JSON format, for the crd resource visitor, we have to take this into consideration, and the yaml package in `apimachinery` provides convenient method to parse YAML or JSON documents.